tests/python_tests/helpers/param_config.py

```diff
@@ -87,20 +90,23 @@ def format_combination_sweep(
         for pack_dst in formats
     ]
 
+class TestParamsConfig(TypedDict):
+    testnames: str
+    formats: Optional[List[FormatConfig]] = None
+    dest_acc: Optional[DestAccumulation] = None
+    approx_mode: Optional[List[str]] = None
+    mathop: Optional[List[str]] = None
+    math_fidelity: Optional[List[int]] = None
+    tile_cnt: Optional[int] = None
```

Comment:
```suggestion
    tile_count: Optional[int] = None
```
How much trouble would it be to fix this finally?

Reply:
Will fix, shouldn't be much trouble